pkg/controller/service/service_controller.go

```diff
@@ -63,6 +62,14 @@ const (
 	notRetryable = false
 
 	doNotRetry = time.Duration(0)
+
+	// LabelNodeRoleMaster specifies that a node is a master
+	// It's copied over to kubeadm until it's merged in core: https://github.com/kubernetes/kubernetes/pull/39112
+	LabelNodeRoleMaster = "node-role.kubernetes.io/master"
+
+	// LabelNodeRoleExcludeBalancer specifies that the node should be
+	// exclude from load balancers created by a cloud provider.
+	LabelNodeRoleExcludeBalancer = "node.role.kubernetes.io/exclude-balancer"
```

Comment:
Seems like these should be part of the API? A labels equivalent of https://github.com/kubernetes/kubernetes/blob/fc1bd98c09e2a91e9926b289ceede6593f96fe86/staging/src/k8s.io/api/core/v1/annotation_key_constants.go

Reply:
I think this needs to be updated with the change from #53678

Reply:
Good catch! Thanks @cblecker